public/static/docs/command-reference/gc.md

```diff
@@ -5,7 +5,7 @@ Remove unused objects from <abbr>cache</abbr> or remote storage.
 ## Synopsis
 
 ```usage
-usage: dvc gc [-h] [-q | -v] [-a] [-T] [-c] [-r REMOTE] [-f] [-j JOBS]
+usage: dvc gc [-h] [-q | -v] [-w] [-a] [-T] [-c] [-r REMOTE] [-f] [-j JOBS]
```

Comment:
minor: let's split into two lines this one (three lines total?), it makes sense at this point (everything after aux options h, q, v can go to the next line)